config.md

```diff
@@ -44,19 +44,19 @@ Project configuration applies to files under its tree (`proj/.clangd` configures
 Configuration is combined when this is sensible. In case of conflicts, user
 config has the highest precedence, then inner project, then outer project.
 
-# Schema
+## Schema
```

Comment:
The problem with bumping Schema to H2, blocks to H3, and attributes within blocks to H4, is we don't have a visually distinct style for H4 headings. Maybe we should! (but definitely its own patch).

Is having a single H1 important somehow?
(I don't have any objection if it looks just as good, but if it looks worse then we need a reason!)

Reply:
Reverted this back. There is no particular "problem" with multiple H1 but it is a convention to only have one (e.g. HTML `<head>`).